src/js/components/useSearchShortcuts.js

```diff
@@ -5,21 +5,31 @@ import {useEffect} from "react"
 import Mousetrap from "mousetrap"
 
 import Tabs from "../state/Tabs"
+import throttle from "lodash/throttle"
 import {adjustSelectedLogIndex} from "../flows/adjustSelectedLogIndex"
 
 export default function() {
   let dispatch = useDispatch()
   useEffect(() => {
-    Mousetrap.bind("down", (e) => {
-      e.preventDefault()
-      dispatch(adjustSelectedLogIndex(1))
-    })
-    Mousetrap.bind("up", (e) => {
+    Mousetrap.bind(
+      "down",
+      throttle((e) => {
+        e.preventDefault()
+        dispatch(adjustSelectedLogIndex(1))
+      }, 200)
```

Comment:
I experimented w a few values here, this seemed to be the fastest that my computer could handle without any odd glitching, and it still feels plenty fast to me